platform/platform-filter-response/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -25,6 +25,9 @@
         <AD description="The HTTP X-XSS-Protection response header is a feature that stops pages from loading when they detect reflected cross-site scripting (XSS) attacks."
             name="X-XSS-Protection" id="xXssProtection" required="true" type="String"
             default="1; mode=block"/>
+        <AD description="The Cache-Control header specifies the behavior for browsers and proxies to cache web resources."
+            name="Cache-Control" id="cacheControl" required="true" type="String"
+            default="Cache-Control: \"public, max-age=604800, immutable\""/>
```

Comment:
Should `public` be the default, or should `private`? Does this need to be in the hardening guide?